client/state/action-types.js

```diff
@@ -573,6 +573,8 @@ export const POST_RESTORE_FAILURE = 'POST_RESTORE_FAILURE';
 export const POST_RESTORE_SUCCESS = 'POST_RESTORE_SUCCESS';
 export const POST_REVISIONS_DIALOG_CLOSE = 'POST_REVISIONS_DIALOG_CLOSE';
 export const POST_REVISIONS_DIALOG_OPEN = 'POST_REVISIONS_DIALOG_OPEN';
+export const POST_REVISIONS_DIFF_VIEW_SPLIT = 'POST_REVISIONS_DIFF_VIEW_SPLIT';
```

Comment:
It sounds super awkward, but we put the view last, so related actions sit next to each other.

This would be 
```
POST_REVISIONS_DIFF_SPLIT_VIEW
POST_REVISIONS_DIFF_UNIFY_VIEW
```